01-express-nodemon/Dockerfile

```diff
@@ -1,15 +1,11 @@
-FROM node:0.10.38
-
-RUN mkdir /src
-
-RUN npm install nodemon -g
-
-WORKDIR /src
-ADD app/package.json /src/package.json
-RUN npm install
-
-ADD app/nodemon.json /src/nodemon.json
-
-EXPOSE 3000
-
-CMD npm start
\ No newline at end of file
+FROM node:latest AS build
+RUN apt-get update && apt-get install -y --no-install-recommends dumb-init
+
+FROM node:18.14.2-bullseye-slim
```

Comment:
**suggestion**: Maybe instead of hard-coding, we could use an `ARG` plus defaults.